### DISCUSSION TITLE
Livewire directives showing on page when used in a different directory structure

### DISCUSSION THREAD

**Discussion by mihai-diaconita** [livewire/livewire#1080] [4. General]
Hi Everyone,

**What seems to be the problem:**
`@livewireScripts`, `@livewireStyles`, and `livewire('hello')` showing in the browser window as plain text.

**Steps to Reproduce:**
Install livewire to a different location rather than the usual Laravel directory setup. 
My application directory: `src/Http/application_name/views/base.blade.php`
Rather than in the default Laravel directory: `resources/views/base.blade.php`

**Are you using the latest version of Livewire:**
`"laravel/framework"`: "^7.8"
`"livewire/livewire"`: "^1.1"

![Screenshot 2020-06-08 at 20 14 16](https://user-images.githubusercontent.com/59828970/84072441-125ea500-a9c7-11ea-8f4d-026c1412c061.png)

- I tried the suggestions from this thread => https://forum.laravel-livewire.com/t/resolved-livewirescripts-and-livewirestyles-showing-on-page/147 => didn't work
- Added the livewire.php config and pointed the asset URL to the correct URL path like so `'asset_url'  => '/application_name',`

I'm not sure how else I could debug this issue as I've tried pretty much everything I could think of.
Installing livewire on a fresh Laravel app everything works as expected.

Any suggestions?

Thanks a lot!

**Comment by nuernbergerA** [comment]
`@livewireScripts` and `@livewireStyles` have to render no matter where your components live.

I assume you are building a package because you use `src`, right?
If so check [this](https://laravel-livewire.com/docs/package-dev) out.
